docs/source/config.rst

```diff
@@ -4,42 +4,48 @@ Configuration File Format
 ``mcy`` relies on a configuration file named ``config.mcy``. There are several sections in an mcy configuration file:
 
 ``[options]``
+-------------
 
 This section contains various configuration options for the mutation generation as well as the tags used.
 
-size <num>
+``size <num>``
 	The number of mutations to be generated.
 
-tags <tagname>..
+``tags <tagname>..``
 	The tags used in the ``[logic]`` and ``[report]`` sections.
 
-seed
-	Random generator seed for reproducible mutation lists.
+``seed``
+	Optional. Random number generator seed, can be set for reproducible mutation lists. (If you wish to know the randomly chosen seed of an existing set of mutations, it can be found in ``database/mutations.ys``.)
 
-select
-	?
+``select <selection>``
+	Optional. Selection of a subset of the design to restrict mutations to. See `yosys -h select <http://www.clifford.at/yosys/cmd_select.html>`_ for a description for the selection pattern format.
+
+	.. note:: The ``select`` keyword here is not the Yosys ``select`` command. The argument ``<selection>`` is used as the optional selection argument to the Yosys ``mutate`` command. While the selection pattern format is identical, you cannot use select subcommands such as ``-module``.
 
 Mutation generation options: for a description of how these values influence the mutation generation algorithm, see :ref:`mutgen`.
 
-weight_cover
-	Default: 500
+``weight_cover``
+	Optional. Default: 500
 
-weight_pq_w weight_pq_b weight_pq_c weight_pq_s weight_pq_mw weight_pq_mb weight_pq_mc weight_pq_ms
-	Default: 100
+``weight_pq_w weight_pq_b weight_pq_c weight_pq_s weight_pq_mw weight_pq_mb weight_pq_mc weight_pq_ms``
```

Comment:
worth putting a link here to the description of what these do (in your other commit)

Reply:
A different link than the one just above on line 25?

Reply:
I think the same link, just repeated. Those terms are all new to me and it wasn't obvious the link in the previous section would also show me what these terms mean.